models/multimeter.h

```diff
@@ -38,10 +38,12 @@
 #include "dictutils.h"
 #include "name.h"
 
-/* BeginUserDocs: device, recorder
+/* BeginUserDocs: device, recorder, continuous
```

Comment:
```suggestion
/* BeginUserDocs: device, recorder
```

Although I know what you want to say, I think `continuous` is not a good tag. IMO, this is more misleading than helpful. Does it mean continuous threshold detection (what we usually call *precise*), spikes on non-grid time points (usually referred to as *off-grid spiking*, measurements of continuous (i.e. non-spike) properties (which is the case here) or, or, or. The term just has too many different meaning to be a good tag.